### PR TITLE
Send target-db-version, MinimumVersionsFixedIn details for assessment/analyze-schema reports to controlplane/callhome

### PR DESCRIPTION
**Pull request by makalaaneesh**
### Describe the changes in this pull request
In assess-migration
- yugabyted control plane 
    - send target-db-version
    - include MinFixVersion for all issues 
- callhome 
    -  send target-db-version
    
In analyze-schema
- yugabyted control plane (Already done because we send the entire schema report)
    - send target-db-version
    - include MinFixVersion for all issues 
- callhome 
    -  send target-db-version

### Does your PR have changes that can cause upgrade issues? 
Changes to callhome and yugabyted. Both are changes to the phase payload, no change in table schema. 
Overall, should not cause upgrade issues. 

| Component        | Breaking changes? |
| :----------------------------------------------: | :-----------: |
| MetaDB                                                    |  Yes/No |
| Name registry json                                        |  Yes/No |
| Data File Descriptor Json                                 |  Yes/No |
| Export Snapshot Status Json                               |  Yes/No |
| Import Data State                                         |  Yes/No |
| Export Status Json                                        |  Yes/No |
| Data .sql files of tables                                 |  Yes/No |
| Export and import data queue                              |  Yes/No |
| Schema Dump                                               |  Yes/No |
| AssessmentDB                                              |  Yes/No |
| Sizing DB                                                 |  Yes/No |
| Migration Assessment Report Json                          |  Yes/No |
| Callhome Json                                             |  No |
| YugabyteD Tables                                          |  No |
| TargetDB Metadata Tables                                  |  Yes/No |

### CODE REVIEW

File: yb-voyager/cmd/common.go [hunk 1]
```diff
@@ -1224,6 +1224,7 @@ type AssessMigrationDBConfig struct {
 type AssessMigrationPayload struct {
 	PayloadVersion        string
```
Comment (priyanshi-yb) [inline yb-voyager/cmd/common.go:1225]: Should we change the payload version as we add the `TargetDBVersion` struct?

File: yb-voyager/src/callhome/diagnostics_test.go [hunk 2]
```diff
@@ -113,8 +115,9 @@ func TestCallhomeStructs(t *testing.T) {
 			name:       "Validate AnalyzePhasePayload Struct Definition",
 			actualType: reflect.TypeOf(AnalyzePhasePayload{}),
 			expectedType: struct {
-				Issues          string `json:"issues"`
-				DatabaseObjects string `json:"database_objects"`
+				TargetDBVersion *ybversion.YBVersion `json:"target_db_version"`
+				Issues          string               `json:"issues"`
+				DatabaseObjects string               `json:"database_objects"`
 			}{},
```
Comment (priyanshi-yb) [inline yb-voyager/src/callhome/diagnostics_test.go:121]: > Callhome Json | Yes

Regarding this, I think this callhome change is not a breaking change as we are just adding new fields to json payload column.

Comment (makalaaneesh) [reply yb-voyager/src/callhome/diagnostics_test.go:121]: Ah, added in a note above saying it's not a breaking change, but forgot to update the table.

**Review by priyanshi-yb — APPROVED** [review]
LGTM

**Pull request merged**